examples/callbacks/callbacks.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/muesli/cache2go"
+	"github.com/trapajim/cache2go"
```

Comment:
Careful, I assume you accidentally committed that change. Please revert it.

Reply:
sorry for that. I've changed it.